docs/code-reviews/process-guidance/reviewer-guidance.md

```diff
@@ -34,6 +34,10 @@ Code reviews should use the below guidance and checklists to ensure positive and
 - Suggest changes to a PR by using the suggestion feature (available in [GitHub](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request) and Azure DevOps) or by creating a PR to the author branch.
 - If one or two comments don’t resolve a disagreement, talk in person or create group discussion.
 
+### Code reviews shouldn't include many lines of code
```

Comment:
I think this fits better in the author guidance than in the reviewer as the author has the power over how big the PRs are

Reply:
In fact, I think the entire section "Make comments clear" can fit in the author side. Gonna make that change and let me know what you think.

Reply:
Looks like this was still not moved